pkg/cmd/release-payload-controller/payload_accepted_controller.go

```diff
@@ -133,9 +139,75 @@ func (c *PayloadAcceptedController) sync(ctx context.Context, key string) error
 		return err
 	}
 
+	originalCondition := conditions.GetReleasePayloadCondition(originalReleasePayload, v1alpha1.ConditionPayloadAccepted)
+	acceptedCondition, ok := computeReleasePayloadAcceptedCondition(originalReleasePayload, originalCondition)
+	if !ok {
+		return nil
+	}
+
+	if !conditions.ConditionChanged(originalCondition, acceptedCondition) {
+		klog.V(5).Infof("ReleasePayload: '%s/%s' already synced", originalReleasePayload.Namespace, originalReleasePayload.Name)
+		return nil
+	}
+
 	releasePayload := originalReleasePayload.DeepCopy()
 
 	klog.V(4).Infof("Syncing Payload Accepted for ReleasePayload: %s/%s", releasePayload.Namespace, releasePayload.Name)
+	conditions.UpdateReleasePayloadCondition(releasePayload, acceptedCondition)
+	releasepayloadhelpers.CanonicalizeReleasePayloadStatus(releasePayload)
+
+	if reflect.DeepEqual(originalReleasePayload, releasePayload) {
+		return nil
+	}
+
+	_, err = c.releasePayloadClient.ReleasePayloads(releasePayload.Namespace).UpdateStatus(ctx, releasePayload, metav1.UpdateOptions{})
+	if errors.IsNotFound(err) {
+		return nil
+	}
+	if err != nil {
+		return err
+	}
 
 	return nil
 }
+
+func computeReleasePayloadAcceptedCondition(payload *v1alpha1.ReleasePayload, originalCondition *metav1.Condition) (*metav1.Condition, bool) {
+	acceptedCondition := &metav1.Condition{
+		Type:               v1alpha1.ConditionPayloadAccepted,
+		Status:             metav1.ConditionUnknown,
+		Reason:             v1alpha1.ReleasePayloadAcceptedReason,
+		LastTransitionTime: metav1.Now(),
```

Comment:
no

Reply:
You want logic like this: https://github.com/openshift/library-go/blob/master/pkg/operator/v1helpers/helpers.go#L54-L72

Reply:
Ack.  Done!